generator/routing_index_generator.cpp

```diff
@@ -373,6 +371,79 @@ void CalcCrossMwmTransitions(
   }
 }
 
+/// \brief Fills |transitions| for experimental transit case. It means that Transition::m_roadMask
+/// for items in |transitions| will be equal to VehicleType::Transit after the call of this method.
+void CalcCrossMwmTransitionsExperimental(
+    string const & mwmFile, string const & intermediateDir, string const & mappingFile,
+    vector<m2::RegionD> const & borders, string const & country,
+    routing::CountryParentNameGetterFn const & /* countryParentNameGetterFn */,
+    vector<routing::CrossMwmConnectorSerializer::Transition<routing::connector::TransitId>> &
+        transitions)
+{
+  CHECK(mappingFile.empty(), ());
+  CHECK(intermediateDir.empty(), ());
+  try
+  {
+    FilesContainerR cont(mwmFile);
+    if (!cont.IsExist(TRANSIT_FILE_TAG))
+    {
+      LOG(LINFO, ("Experimental transit cross mwm section is not generated because there is no "
+                  "experimental transit section in mwm:",
+                  mwmFile));
+      return;
+    }
+    auto reader = cont.GetReader(TRANSIT_FILE_TAG);
+
+    transit::experimental::TransitData transitData;
+    transitData.DeserializeForCrossMwm(*reader.GetPtr());
+    auto const & stops = transitData.GetStops();
+    auto const & edges = transitData.GetEdges();
+
+    auto const getStopIdPoint = [&stops](transit::TransitId stopId) {
+      auto const it = find_if(
+          stops.begin(), stops.end(),
+          [stopId](transit::experimental::Stop const & stop) { return stop.GetId() == stopId; });
+
+      CHECK(it != stops.end(), ());
```

Comment:
`CHECK(it != stops.end(), ("stopId:", stopId, "is not found in stops. Size of stops:", stops.size()));`?